test/bin/ci_phase_iso_build.sh

```diff
@@ -76,16 +76,9 @@ update_build_cache() {
 # - Always build the 'presubmit' layer.
 # - Only build the 'periodic' layer when 'CI_JOB_NAME' contains 'periodic' token.
 run_image_build() {
-    local -r with_cached_data=$1
-
     if [ -v CI_JOB_NAME ] ; then
         # Conditional per-layer builds when running in CI
-        if ${with_cached_data} ; then
-            $(dry_run) bash -x ./bin/build_images.sh -d -l ./image-blueprints/layer1-base
-        else
-            $(dry_run) bash -x ./bin/build_images.sh    -l ./image-blueprints/layer1-base
-        fi
-
+        $(dry_run) bash -x ./bin/build_images.sh -l ./image-blueprints/layer1-base
```

Comment:
Can we add a comment here explaining how / when the cache is used?

Reply:
done